File: app/src/main/java/com/yjz/notepad/MainActivity.java
package com.yjz.notepad;

import android.os.Handler;

import com.alibaba.android.arouter.launcher.ARouter;
import com.seabig.common.base.BaseActivity;


public class MainActivity extends BaseActivity {

    @Override
    protected boolean onSettingFullScreen() {
        return true;
    }

    @Override
    protected int onSettingUpContentViewResourceID() {
        return R.layout.activity_main;
    }

    @Override
    protected void onSettingUpView() {
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                ARouter.getInstance().build("/load/activity/register").navigation();
                finish();
            }
        }, 1000);
    }
}

File: bookkeeping/src/main/java/com/yjz/bookkeeping/ui/activity/MainActivity.kt
package com.yjz.bookkeeping.ui.activity

import com.seabig.common.base.BaseActivity
import com.yjz.bookkeeping.R
import com.yjz.bookkeeping.ui.fragment.BookkeepingFragment

/**
 * author： YJZ
 * date:  2018/11/26
 * des:  首页
 */

class MainActivity : BaseActivity() {

    override fun onSettingUpContentViewResourceID(): Int {
        return R.layout.bookkeeping_fragment_main
    }

    override fun onSettingUpView() {
        supportFragmentManager
                .beginTransaction()
                // 此处的R.id.fragment_container是要盛放fragment的父容器
                .add(R.id.fragment_container, BookkeepingFragment())
                .commit()
    }
}
